code/harry_potter_store/test/test_store.py

```diff
@@ -4,72 +4,58 @@
+        self.assertEqual(8, self.store.get_price({"1": 1}))
 
     def test_get_price_2(self):
-        store = Store()
-        self.assertEqual(store.get_price({"1": 2}), 16)
+        self.assertEqual(16, self.store.get_price({"1": 2}))
 
     def test_get_price_with_discount_2_books(self):
-        store = Store()
-        self.assertEqual(store.get_price({"1": 1, "2": 1}), 16 * 0.95)
+        self.assertEqual(16 * 0.95, self.store.get_price({"1": 1, "2": 1}))
 
     def test_get_price_with_discount_3_books(self):
-        store = Store()
-        self.assertEqual(store.get_price({"1": 1, "2": 1, "3": 1}), 24 * 0.9)
+        self.assertEqual(24 * 0.9, self.store.get_price({"1": 1, "2": 1, "3": 1}))
 
     def test_get_price_with_discount_4_books(self):
-        store = Store()
-        self.assertEqual(store.get_price({"1": 1, "2": 1, "3": 1, "4": 1}), 32 * 0.8)
+        self.assertEqual(32 * 0.8, self.store.get_price({"1": 1, "2": 1, "3": 1, "4": 1}))
 
     def test_get_price_with_discount_5_books(self):
-        store = Store()
-        self.assertEqual(store.get_price({"1": 1, "2": 1, "3": 1, "4": 1, "5": 1}), 40 * 0.75)
+        self.assertEqual(40 * 0.75, self.store.get_price({"1": 1, "2": 1, "3": 1, "4": 1, "5": 1}))
 
     def test_get_price_complex_1(self):
-        store = Store()
-        self.assertEqual(store.get_price({"1": 1, "2": 2}), 16 * 0.95 + 8)
+        self.assertEqual(16 * 0.95 + 8, self.store.get_price({"1": 1, "2": 2}))
 
     def test_get_price_complex_2(self):
-        store = Store()
-        self.assertEqual(store.get_price({"1": 0, "2": 2}), 16)
+        self.assertRaises(ValueError, lambda: self.store.get_price({"1": 0, "2": 2}))
 
     def test_get_price_complex_3(self):
-        store = Store()
-        self.assertEqual(store.get_price({"1": -132, "2": -3324}), 0)
+        self.assertRaises(ValueError, lambda: self.store.get_price({"1": -132, "2": -3324}))
 
     def test_get_price_complex_4(self):
-        store = Store()
-        self.assertEqual(store.get_price({"1": -132, "2": -3324, "111": 333}), 0)
+        self.assertRaises(ValueError, lambda: self.store.get_price({"1": -132, "2": -3324, "111": 333}))
 
     def test_get_price_complex_5(self):
-        store = Store()
-        self.assertEqual(store.get_price({"1": 3, "2": 1, "3": 2}), 24 * 0.90 + 16 * 0.95 + 8)
+        self.assertEqual(24 * 0.90 + 16 * 0.95 + 8, self.store.get_price({"1": 3, "2": 1, "3": 2}))
 
     def test_get_price_complex_6(self):
-        store = Store()
-        self.assertEqual(store.get_price({"1": 2, "2": 2, "3": 2, "4": 2}), (32 * 0.8) * 2)
+        self.assertEqual((32 * 0.8) * 2, self.store.get_price({"1": 2, "2": 2, "3": 2, "4": 2}))
 
     def test_get_price_complex_7(self):
-        store = Store()
-        self.assertEqual(store.get_price({"1": 2, "2": 2, "3": 2, "4": 1, "5": 1}),
-                         (40 * 0.75) + (24 * 0.9))
+        self.assertEqual(32 * 0.8 * 2,
```

Comment:
Please check following test case:
`{"1": 2, "2": 1, "3": 2, "4": 1, "5": 2}`

Reply:
This was not a random request, but a bug in the code.

Are you sure that 
`{"1": 2, "2": 2, "3": 2, "4": 1, "5": 1})` (case 7) and
`{"1": 2, "2": 1, "3": 2, "4": 1, "5": 2}` (case 10)
should give different results?

Reply:
add sorting